### PR TITLE
Input file

### PR DESCRIPTION
**Pull request by ester-agishtein**

### CODE REVIEW

File: src/main/java/spof/FileReader.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package spof;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileReader {
```
Comment (rpeikes) [inline src/main/java/spof/FileReader.java:9]: Is FileReader a class in the java API? If so, let's change the name to GraphReader or something

File: src/main/java/spof/FileReader.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    public List<List<String>> getNetworks() {
+        return networks;
+    }
+
+
+
+    public FileReader(String filePath) {
+        try {
+            File myObj = new File(filePath);
```
Comment (rpeikes) [inline src/main/java/spof/FileReader.java:21]: maybe rename this as file or anything a little more specific?

File: src/main/java/spof/FileReader.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            Scanner myReader = new Scanner(myObj);
+            List<String> network = new ArrayList<>();
+            while (myReader.hasNextLine()) {
+
+                String data = myReader.nextLine();
+
+                if(data.compareTo("0") == 0) {
```
Comment (rpeikes) [inline src/main/java/spof/FileReader.java:28]: if(data.equals("0"))

might be more straight forward

**Review by rpeikes — COMMENTED** [review]
Looks awesome overall! 
Let's go through it together so we can all understand what everything does

**Review by shweinfeld — COMMENTED** [review]
Looking good!

File: src/main/java/spof/FileReader.java (hunk 1 above)
Comment (shweinfeld) [reply src/main/java/spof/FileReader.java:9]: Agreed

**Pull request merged**